integration/cli-tests/src/test/java/EnvSetup.sh

```diff
@@ -44,7 +44,8 @@ cd $BASEDIR
 DIR="../../../../../cmd/build"
 
 if [ -d "$DIR" ]; then
-    echo "CLI build exists. Hence skipping the environment setup phase"
+    echo "CLI build exists. Extracting Compress Archive"
+    extractCompressArchive
```

Comment:
shall we move this also outside if block, since it is there in if and else path

Reply:
Done. Please review the latest commit